docs/main/prerequisites.md

```diff
@@ -26,6 +30,13 @@ The following operating systems are **officially** supported:
 | Fedora       | 31 / 32          | ARM / x86_64        |
 | CentOS       | 7 / 8            | x86_64              |
 
+!!! info
+    Pi-hole will run on most Debian-based distributions and is the preferred platform.
+
+!!! info
+    For Fedora-based distributions, it is advisable to set [SELinux to `disabled` or `permissive`](https://wiki.centos.org/HowTos/SELinux#SELinux_Modes).
```

Comment:
We don't advise to make changes to SELinux. I don't think that is advisable.

Reply:
Weird, I recently tried Pi-hole with SELinux in `enforcing`, and the Admin panel (I think that is what it's called) wouldn't show anything

Reply:
That may be the case but we don't advise people to disable their systems security features. We don't want people to set SEL to a disabled state without knowing what/why they are doing it.

Reply:
Makes sense

Reply:
Based on this https://github.com/pi-hole/pi-hole/issues/752#issuecomment-252159257, it seems like it installs the policy automatically, I could be wrong though

Reply:
Those are commits from October of 2016. A lot has changed since then.

Reply:
True

Reply:
@bcambl Thoughts on how to properly word this?

Reply:
So... we have to wait for him to answer?

Reply:
Yes.

Reply:
@dschaper so, i just tried setting up Pi-hole in Fedora with SELinux in `enforcing`, and i got this:

```
[root@localhost themainvm]# curl -sSL https://install.pi-hole.net | bash

  [✓] Root user check

        .;;,.
        .ccccc:,.
         :cccclll:.      ..,,
          :ccccclll.   ;ooodc
           'ccll:;ll .oooodc
             .;cll.;;looo:.
                 .. ','.
                .',,,,,,'.
              .',,,,,,,,,,.
            .',,,,,,,,,,,,....
          ....''',,,,,,,'.......
        .........  ....  .........
        ..........      ..........
        ..........      ..........
        .........  ....  .........
          ........,,,,,,,'......
            ....',,,,,,,,,,,,.
               .',,,,,,,,,'.
                .',,,,,,'.
                  ..'''.

  [✓] Disk space check
  [✓] Update local cache of available packages

  [✓] Checking dnf for upgraded packages... 473 updates available
  [i] It is recommended to update your OS after installing the Pi-hole!

  [i] Installer Dependency checks...
  [✓] Checking for git
  [✓] Checking for iproute
  [✓] Checking for newt
  [✓] Checking for procps-ng
  [✓] Checking for which
  [✓] Checking for chkconfig

  [✗] Default SELinux: enforcing
  [✗] Current SELinux: Enforcing
  Pi-hole does not provide an SELinux policy as the required changes modify the security of your system.
  Please refer to https://wiki.centos.org/HowTos/SELinux if SELinux is required for your deployment.

  SELinux Enforcing detected, exiting installer

```

do we still not recommend to set SELinux to `permissive`/`disabled` or do we keep it unchanged?

Reply:
I don't think Pi-hole needs to be recommending the disabling of a system-wide security feature. The warning displayed is a nice way of saying "You're on your own." If you're installing a DNS server on an operating system that is SELinux on by default then you should know the risks/rewards for changing the defaults. 

@pi-hole/core-approvers can chime in with their thoughts.

Reply:
alright, also thank you for the REALLY quick responses

Reply:
Thank you for the compliment! I think we're set to approve and merge, I don't think there's any objections.